cfpb/capital-framework
# Replace uglifyjs-webpack-plugin with terser-webpack-plugin

See background guidance here https://github.com/webpack/webpack/issues/7923

## Changes

- Switch to `terser-webpack-plugin` from `uglifyjs-webpack-plugin`.
- Updates to latest `webpack`.
- Regenerates yarn.lock.

## Testing

1. Run `yarn`
2. Run `gulp build && gulp docs && bundle exec jekyll serve` and check that the docs are correct.
